doc/CMakeLists.txt

```diff
@@ -31,7 +31,8 @@
 
 find_program(PANDOC pandoc)
 if (NOT PANDOC)
-        message(WARNING "pandoc not found")
+        message(FATAL
+	    "pandoc not found, needed for generating manpages -- set WITH_INSTALL_TARGETS to OFF to disable manpage generation")
```

Comment:
Seemed to make sense with the WITH_INSTALL_TARGETS option.
But since it wouldn't allow installing without up to date manpage anyways, I reverted this.